frame/nomination-pools/src/tests.rs

```diff
@@ -4198,6 +4210,44 @@ mod create {
 			);
 		});
 	}
+
+	#[test]
+	fn create_with_pool_id_works() {
+		ExtBuilder::default().build_and_execute(|| {
+			let ed = Balances::minimum_balance();
+
+			Balances::make_free_balance_be(&11, StakingMock::minimum_bond() + ed);
+			assert_ok!(Pools::create(
+				RuntimeOrigin::signed(11),
+				StakingMock::minimum_bond(),
+				123,
+				456,
+				789
+			));
```

Comment:
There are accounts created in mock.rs that are already funded. You need not create a new one. 

also, I think this test will pass entirely without you creating this new pool as well, not sure what you are testing here.